README.md

```diff
@@ -27,7 +27,8 @@ Authentication details for the Azure validator controller are provided within ea
   * To use this method:
     1. Set Helm value `AzureValidator.auth.implicit` to `true`.
     1. Ensure workload identity is set up for your AKS cluster, including the [managed identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-a-managed-identity) and [federated identity credential](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#establish-federated-identity-credential).
-    1. Create a Kubernetes ServiceAccount for use with the plugin that is [configured appropriately for workload identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-kubernetes-service-account) and set Helm value `AzureValidator.auth.serviceAccountName` to the name of this ServiceAccount.
+    1. Create a Kubernetes ServiceAccount for use with the plugin that is [configured appropriately for workload identity](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#create-kubernetes-service-account). The ServiceAccount must include all RBAC privileges found in `templates/manager-rbac.yaml`. Set Helm value `AzureValidator.auth.serviceAccountName` to the name of this ServiceAccount.
```

Comment:
The required RBAC privileges should be a link to https://github.com/spectrocloud-labs/validator-plugin-azure/blob/main/chart/validator-plugin-azure/templates/manager-rbac.yaml